pkg/api/types.go

```diff
@@ -1824,6 +1824,9 @@ type PodSpec struct {
 	// If not specified, the pod will not have a domainname at all.
 	// +optional
 	Subdomain string
+	// If specified, the pod's scheduling preferences
```

Comment:
"Scheduling preferences" is too vague. Also preferences doesn't imply that it can be hard constraints or soft preferences.